grpc.gyp

```diff
@@ -62,6 +62,10 @@
       '-Isrc/core/ext/upbdefs-gen',
```

Comment:
We can probably delete this whole file, right? JS is no longer supported in this repo.

Reply:
Yep, I think we can. https://github.com/grpc/grpc/pull/36077